src/hansel.mjs

```diff
@@ -1,8 +1,34 @@
-import { findElementWithHandler, toArray, warn } from './util';
+import {
+  findElementWithHandler,
+  isAnchor,
+  isModifierKey,
+  toArray,
+  warn,
+} from './util';
 import { ENHANCER_ATTRIBUTE, HANDLER_ATTRIBUTE } from './constants';
 
 /**
- * enhance :: DomNode -> Object -> Array
+ * Get handler function for given handler.
+ */
+const getHandlerFn = handler => {
+  if (typeof handler === 'function') {
+    return handler;
+  }
+  if (handler && handler.fn) {
+    return handler.fn;
+  }
+  return undefined;
+};
+
+/**
+ * Get optional handler options.
+ */
+const getHandlerOptions = handler => handler && handler.options
+  ? handler.options
+  : {};
```

Comment:
I'm wondering here if we should make this the default options object:

```js
{
  allowModifierKeys: false
}
```

Just to get an overview of the defaults. 
Not sure, definitely no requirement right now.

Reply:
Yeah, good catch! In the previous version it was already there, but due to the backwards compatibility it's not an argument to a function anymore (with defaults). But I'll add a default which will mix it with passed options.

Reply:
Something like this: https://github.com/grrr-amsterdam/hansel/pull/14/files#diff-c9301d89ef2ddba8638ef0b69b6e7979R30

Reply:
Yes, that's very clear! 👍